hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeStateMachine.java

```diff
@@ -262,6 +262,11 @@ private void start() throws IOException {
     }
   }
 
+  public void handleFatalVolumeFailures() {
+    LOG.error("DatanodeStateMachine Shutdown due to fatal volume failures");
```

Comment:
A more detail error message is preferred here since we know the right reason.

Reply:
sure, I'll put a more clear message here.